src/app/branded/branded-checkout.component.js

```diff
@@ -187,6 +187,6 @@ export default angular
       onOrderFailed: '&',
       language: '@',
       showCoverFees: '@',
-      useV3: '@'
+      useV3: '@',
```

Comment:
This commit should just go away since the only change is removed in the next commit.